bokehjs/test/unit/models/text/utils.ts

```diff
@@ -0,0 +1,32 @@
+import {expect} from "assertions"
+
+import {is_tex_string} from "@bokehjs/models/text/utils"
+
+describe("Text utils", () => {
+  it("is_tex_string", async () => {
+    expect(is_tex_string("$$test$$")).to.be.true
+    expect(is_tex_string("\\[test\\]")).to.be.true
+    expect(is_tex_string("\\(test\\)")).to.be.true
+    expect(is_tex_string("HTML <b>text</b> $$\\sin(x) and \\[x\\cdot\\pi\\]!")).to.be.false
+    expect(is_tex_string("\\[test\\]")).to.be.true
+    expect(is_tex_string("\\(test\\)")).to.be.true
+    expect(is_tex_string("test$$")).to.be.false
+    expect(is_tex_string("$$test")).to.be.false
+    expect(is_tex_string("HTML <b>text</b> $$sin(x)$$ and [xcdotpi]!")).to.be.false
+    expect(is_tex_string("$$test\\]")).to.be.false
+    expect(is_tex_string("$$test $$ end $$")).to.be.true
+    expect(is_tex_string("$$ \\[test end\\]")).to.be.false
+    expect(is_tex_string("text \\[text $$latex$$")).to.be.false
+    expect(is_tex_string("$$ tex [ tex ] tex $$")).to.be.true
+    expect(is_tex_string("$$tex$$text$$tex$$")).to.be.true
+    expect(is_tex_string("part0$$part1\\[part2\\(part3$$")).to.be.false
+    expect(is_tex_string("part0$$part1\\[part2\\(part3\\]")).to.be.false
+    expect(is_tex_string("part0$$part1\\[part2\\(part3\\)")).to.be.false
+    expect(is_tex_string(`$$
+      cos(x)
+    $$`)).to.be.true
+    expect(is_tex_string(`$$
+      cos(x)$$
+    `)).to.be.false
+  })
```

Comment:
For future reference, structure such test at least a little bit, taking advantage that you can nest `describe` clauses, e.g.:
```ts
describe("models/text/utils module", () => {
  describe("provides is_text_string() function", () => {
    it("that supports multi-line inputs", () => ...
```
Tests fail on the first failed `except`, so the fewer in an `it`, the better (with reason obviously).